browser-tests/pages/register.ts

```diff
@@ -0,0 +1,34 @@
+import { screen } from '@testing-library/testcafe';
+import { Selector } from 'testcafe';
+
+export const registerForm = {
```

Comment:
instead of simple const object, I would rather use class based pagemodel which is recommended by testcafe: https://github.com/DevExpress/testcafe-examples/blob/master/examples/use-page-model/page-model.js
You can also add actions there as methods like clicking single button or even longer operation like filling form and submit.

Reply:
Keep this as it is now, class based pagemodel will use with new tests